File: Iniciante/1041 - Coordenadas de um Ponto.py
"""
Autor: Pierre Vieira
Data da submissão: 25/01/2019 01:09:01
"""
linha = input().split(" ")
X, Y = linha
x = float(X)
y = float(Y)
if x == 0 and y == 0:
    print("Origem")
elif x == 0 and y !=0:
    print("Eixo Y")
elif x != 0 and y == 0:
    print("Eixo X")
elif x > 0 and y >0:
    print("Q1")
elif x > 0 and y < 0:
    print("Q4")
elif x <0 and y < 0:
    print("Q3")
else:
    print("Q2")

File: C#/Iniciante/1041 - Coordenadas de um Ponto.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 11:41:01
*/
using System;
using System.Globalization;
using Microsoft.VisualBasic;

namespace URI {
    class URI {
        static void Main(string[] args)
        {
            string[] linha = Console.ReadLine().Split(' ');
            double x, y;
            x = double.Parse(linha[0]);
            y = double.Parse(linha[1]);
            if (x == 0 && y == 0)
            {
                Console.WriteLine("Origem");
            }
            else if (x == 0)
            {
                Console.WriteLine("Eixo Y");
            }
            else if (y == 0)
            {
                Console.WriteLine("Eixo X");
            }
            else if (x < 0 && y < 0)
            {
                Console.WriteLine("Q3");
            }
            else if(x > 0 && y > 0)
            {
                Console.WriteLine("Q1");
            }
            else if (x < 0 && y > 0)
            {
                Console.WriteLine("Q2");
            }
            else
            {
                Console.WriteLine("Q4");
            }
        }
    }
}
